_getting-started/security.md

```diff
@@ -0,0 +1,192 @@
+
+You can find complete details regarding demo configuration and how to get up and running quickly at [Setting up a demo configuration]({{site.url}}{{site.baseurl}}/security/configuration/demo-configuration/)
+{: .note}
+
+Certain aspects of this configuration, such as demo certificates and default passwords, should never be used in production. These parts of the demo configuration should be updated with your custom information before proceeding to production.
+{: .warning}
+
+## Setting up the demo configuration
+
+Prior to running the `install_demo_configuration.sh` script you must create environment variable named `OPENSEARCH_INITIAL_ADMIN_PASSWORD` set to a strong password. This will be used as the password for the admin user to authenticate with OpenSearch. Use the online tool [_Zxcvbn_](https://lowe.github.io/tryzxcvbn/) to test the strength of any password. After this is set, you can execute `install_demo_configuration.sh` and follow the terminal prompt to enter necessary details.
+
+After the script is executed, you can start OpenSearch and test out the configuration by running the following command:
+
+```
+curl -k -XGET -u admin:<password> https://<opensearch-ip>:9200
+```
+{% include copy.html %}
+
+You should see similar output to the following:
+
+```
+{
+  "name" : "smoketestnode",
+  "cluster_name" : "opensearch",
+  "cluster_uuid" : "0a5DYAk0Rbi14wqT3TqMiQ",
+  "version" : {
+    "distribution" : "opensearch",
+    "number" : "2.13.0",
+    "build_type" : "tar",
+    "build_hash" : "7ec678d1b7c87d6e779fdef94e33623e1f1e2647",
+    "build_date" : "2024-03-26T00:04:51.025238748Z",
+    "build_snapshot" : false,
+    "lucene_version" : "9.10.0",
+    "minimum_wire_compatibility_version" : "7.10.0",
+    "minimum_index_compatibility_version" : "7.0.0"
+  },
+  "tagline" : "The OpenSearch Project: https://opensearch.org/"
+}
+```
+
+## Setting up OpenSearch Dashboards
+
+In order to quickly get started with OpenSearch Dashboards, you can add the following configuration to `opensearch_dashboards.yml`:
+
+```
+opensearch.hosts: [https://localhost:9200]
+opensearch.ssl.verificationMode: none
+opensearch.username: kibanaserver
+opensearch.password: kibanaserver
+opensearch.requestHeadersWhitelist: [authorization, securitytenant]
+
+opensearch_security.multitenancy.enabled: true
+opensearch_security.multitenancy.tenants.preferred: [Private, Global]
+opensearch_security.readonly_mode.roles: [kibana_read_only]
+# Use this setting if you are running opensearch-dashboards without https
+opensearch_security.cookie.secure: false
+```
+{% include copy.html %}
+
+You can start the binary or service, depending on which method was used to install OpenSearch and OpenSearch Dashboards.
+
+When using binary, you need to supply `--no-base-path` to `yarn start` command to set a URL without a base-path. If this is not set, a random 3-letter base-path will be added.
```

Comment:
Above we use "the binary". Should "the" precede "binary" here? Should there also be an article preceding `--no-base-path`?